src/Microsoft.ML.AutoML/Experiment/MetricsAgents/BinaryMetricsAgent.cs

```diff
@@ -78,7 +78,7 @@ public bool IsModelPerfect(double score)
             }
         }
 
-        public BinaryClassificationMetrics EvaluateMetrics(IDataView data, string labelColumn)
+        public BinaryClassificationMetrics EvaluateMetrics(IDataView data, string labelColumn, string groupId)
```

Comment:
>groupId [](start = 102, length = 7)

nit: If this represents a column, should the name be groupIdColumn to match the others?